src/components/TaskList.js

```diff
@@ -1,34 +1,46 @@
 import styled from 'styled-components';
 
-export default function TaskList({todos, onTodoChange}) {
+export default function TaskList({todos, onTodoChange, deleteTodo}) {
   return todos.map(todo => {
     return (
       <Wrapper key={todo.id}>
         <StyledTitle>{todo.title}</StyledTitle>
-
-        <p>{todo.description}</p>
-
+        <Description>{todo.description}</Description>
         {todo.tasks.length === 0 ? (
           <Image src={todo.image} alt="funny dog" />
         ) : (
-          <StyledList role="list">
+          <ul>
             {todo.tasks.map(task => {
               return (
-                <ListItem key={task.id}>
-                  <label>
-                    <Checkbox
-                      type="checkbox"
-                      checked={task.isChecked}
-                      data-todoid={todo.id}
-                      data-taskid={task.id}
-                      onChange={onTodoChange}
-                    />
-                    <Task checked={task.isChecked}>{task.task}</Task>
-                  </label>
-                </ListItem>
+                <div key={task.id}>
+                  <ListItem>

```

Comment:
You don't need the `div` here, you can simply add the key to your `ListItem`.

Reply:
On line 39 there is a `<hr/>`, on the same level with the ListItem. I need the div here as a parent element, because otherwise react gives me "Each child in a list should have a unique "key" prop". Or? :/

Reply:
Didn't see that, fine!